app/parse.py

```diff
@@ -8,8 +15,53 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def parse_single_quote(quote) -> Quote:
+    return Quote(
+        text=quote.select_one(".text").text,
+        author=quote.select_one(".author").text,
+        tags=[tag.text for tag in quote.select(".tag")],
+    )
+
+
+def get_single_page_quotes(page_soup) -> [Quote]:
```

Comment:
The return type hint for `get_single_page_quotes` should be `List[Quote]` instead of `[Quote]`. You need to import `List` from the `typing` module to use it as a type hint.